### ISSUE TITLE
tex option --shell-escape not working

### ISSUE THREAD

**Issue by wangzhengbeijian** [SublimeText/LaTeXTools#861]
I use package minted to compile my tex, which requires shell-escape option. So I add that directive at the top of file, but it doesn't work.

``` latex
%!TEX program = xelatex
%!TEX options = --shell-escape
\documentclass{article}
\usepackage{minted}
\begin{document}
\end{document}
```

And finally I find the solution. At line 128 at file 'tradiotionalBuilder.py', I change \" to ', and it works. 

``` python
for option in self.options:
    if texify:
        cmd.append(u"--tex-option=\"" + option + "\"")
    else:
        cmd.append(u"-latexoption='" + option + "'")
```

I'm not familiar with python, and don't know why. Hope you can do some change to tradiotionalBuilder.py to solve the problem, may not according to my solution.

**Comment by ig0774** [comment]
Thanks for pointing this out. I thought this was working already, but clearly I missed something. What OS are you on?

**Comment by wangzhengbeijian** [comment]
Windows 7, Sublime Text 3126

**Comment by ig0774** [comment]
This has been fixed in the most recent release.

**Comment by Sharparam** [comment]
I also noticed the `%!TeX options = ...` not working properly recently. The fix that was implemented didnt' fix it for me though. If I remove the quoting entirely, it works. So making it look like this:

```
for option in self.options:
  if texify:
    cmd.append(u"--tex-option=\"" + option + "\"")
  else:
    cmd.append(u"-latexoption=" + option + "")
```

With the quotes in, latexmk parses it as a file and complains about not finding it: "I can't find file '-shell-escape'".

I'm not sure if the quotes are required for other types of options, but for `-shell-escape`, quoting it breaks it.

**Comment by Gitsack** [comment]
This is still not working.

```Latexmk: applying rule 'pdflatex'...
This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016/W32TeX) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
! I can't find file `'--shell-escape''.
<*> '--shell-escape' 
```

How can I fix this?

**Comment by Sharparam** [comment]
@Gitsack See my comment above yours, you have to make it not quote the options.

I haven't made a PR to change this as I'm unsure if the quoting might be necessary for other options.

**Comment by Gitsack** [comment]
@Sharparam thanks for your help, it works now.

I tried it before with your changes but it threw an error then about not finding pdflatex or something and I changed it back and tried other things.

Don't know what I did differently now but it works! Thanks! :)

**Comment by ig0774** [comment]
This should finally be fixed in v3.12.12.
